packages/api/src/controllers/sensorController.js

```diff
@@ -0,0 +1,319 @@
+/*
```

Comment:
This file has a number of console.log's that should be (eventually?) removed.
More important, there are a number of places where it calls a model's `query` method, or similar knex/Objection calls. Please refactor to avoid this by moving the code into the model behind a custom function.

Reply:
@Cali0707

Reply:
I believe I refactored this quite a bit this in a different pull request. It refactors database queries into the models, and removes the console logs, we can leave this for now, and take a look at in that PR after this is merged.